sdk/servicebus/service-bus/src/receivers/receiver.ts

```diff
@@ -375,7 +389,9 @@ export class ReceiverImpl<ReceivedMessageT extends ReceivedMessage | ReceivedMes
 
     const deferredMessages = await this._context.managementClient!.receiveDeferredMessages(
       sequenceNumbers,
-      convertToInternalReceiveMode(this.receiveMode)
+      convertToInternalReceiveMode(this.receiveMode),
+      undefined,
```

Comment:
Ha, the dreaded "undefined" so I can pass options ;)

(it's okay, it's internal).